distribution/docker/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN --mount=type=cache,target=/root/.m2 if [ "$BUILD_FROM_SOURCE" = "true" ]; th
       install \
       -Pdist,bundle-contrib-exts \
       -Pskip-static-checks,skip-tests \
-      -Dmaven.javadoc.skip=true -T1C \
+      -Dmaven.javadoc.skip=true -T1C -X \
```

Comment:
I don't think we should use the -X flag in production by default.
We should leave it to admin to enable debugging if require.

Reply:
Yes, I am actually trying to debug the failing `Distributing Checks` check, added it for that purpose. Will revert it eventually.

My bad, should've dropped a message to inform others looped into this PR.

Reply:
This change has been reverted, hence resolving this conversation.